builtin/common/vector.lua

```diff
@@ -138,3 +138,8 @@ function vector.divide(a, b)
 			z = a.z / b}
 	end
 end
+
+function vector.sort(a, b)
+	return {x = math.min(a.x, b.x), y = math.min(a.y, b.y), z = math.min(a.z, b.z)},
+		{x = math.max(a.x, b.x), y = math.max(a.y, b.y), z = math.max(a.z, b.z)}
+end
```

Comment:
Nice function, but confusing name. Something like `minmax` may be better (I know a library that call that `AABB`... dunno why, but sounds somewhat related to the bounding box that function is used for).